src/vivarium/framework/artifact/artifact.py

```diff
@@ -214,15 +214,15 @@ def __str__(self):
         return out
 
 
-def _to_tree(keys: List[str]) -> Dict[str, Dict[str, List[str]]]:
-    out = defaultdict(lambda: defaultdict(list))
+def _to_tree(keys: list[str]) -> dict[str, dict[str, list[str]]]:
```

Comment:
Add return type - I'm surprised mypy passed?

Reply:
It is there?